modules/nw-configuring-qinq-sriov-proc.adoc

```diff
@@ -0,0 +1,243 @@
+
+* You installed the OpenShift CLI (`oc``).
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the SR-IOV Network Operator.
+
+.Procedure
+
+. To verify that the SR-IOV Operator is ready, run the following command and view the resulting output:
++
+[source,terminal]
+----
+$ oc get csv -n openshift-sriov-network-operator
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME                                          DISPLAY                   VERSION               REPLACES   PHASE
+sriov-network-operator.v4.15.0-202401261531   SR-IOV Network Operator   4.15.0-202401261531              Succeeded
+----
+
+. To verify that the SR-IOV pods are deployed, run the following command:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-sriov-network-operator
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME                                      READY   STATUS    RESTARTS   AGE
+network-resources-injector-drcmj          1/1     Running   0          9m37s
+operator-webhook-5s5vd                    1/1     Running   0          9m37s
+sriov-network-config-daemon-zv2lj         1/1     Running   0          9m37s
+sriov-network-operator-6c5dbb89c8-lmd5s   1/1     Running   0          9m53s
+----
+
+. Check the machine config pool:
++
+[source,terminal]
+----
+$ oc get mcp
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME         CONFIG                                                 UPDATED   UPDATING   DEGRADED   MACHINECOUNT   READYMACHINECOUNT   UPDATEDMACHINECOUNT   DEGRADEDMACHINECOUNT   AGE
+master       rendered-master-acd1358917e9f98cbdb599aea622d78b       True      False      False      3              3                   3                     0                      22h
+worker-cnf   rendered-worker-cnf-1d871ac76e1951d32b2fe92369879826   False     True       False      2              1                   1                     0                      22h
+----
+
+. Create an `SriovNetworkNodePolicy` object, and then save the YAML in the `<name>-sriov-node-network.yaml` file. Replace <name> with the name for this configuration.
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: sriovnetpolicy-810
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: netdevice
+  nicSelector:
+    pfNames:
+      - ens5f0#0-9
+  nodeSelector:
+    node-role.kubernetes.io/worker-cnf: ""
+  numVfs: 10
+  priority: 99
+  resourceName: resource810
+----
+
+. Optional: Label the SR-IOV capable cluster nodes with `SriovNetworkNodePolicy.Spec.NodeSelector` if they are not already labeled. For more information about labeling nodes, see "Understanding how to update labels on nodes".
+
+. Create the `SriovNetworkNodePolicy` object:
++
+[source,terminal]
+----
+$ oc create -f <name>-sriov-node-network.yaml
+----
++
+After applying the configuration update, all the pods in `sriov-network-operator` namespace move to the `Running` status.
```

Comment:
let's not add verification for these steps. Let's keep the same format as in other pages of the sriov documentation.